### PR TITLE
Faq component page

### PR DESCRIPTION
**Pull request by nickstaylor**
# PR Description

Please include a summary of the change:

In this PR, we added the FAQ page with styling, added an Error component that redirects users back home. The error component has completed testing.  New routes were added in the app, and a fetch call was added to the Home component in order to retrieve the FAQ's and place them in the redux store.  Methods and tests are added to the redux actions and reducers to account for the new FAQ array in state.  Minor styling fixes on other components are included as well.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.

New Features:  FAQ page with fetch call and styling, also Error route added with styling and testing.

Note any questions or struggles as well.

## Type of change
Please uncheck options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Add Testing
- [ ] Refactor

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/containers/Error/Error.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            </Link >
+        </div>
+    )
+}
+
+const mapDispatchToProps = (dispatch) => ({
+  clearAllData: () => dispatch(actions.clearAllData()),
+})
+
+export default connect(null, mapDispatchToProps)(Error)
```
Comment (alex-latham) [inline src/containers/Error/Error.js:22]: Thank you so much for this error component! I think it really helps the app shine with regard to error handling

File: src/containers/Faq/Faq.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+@media screen and (max-width: 700px) {
+.faq-header {
+  flex-direction: column;
+  align-items: center;
+}
+.faq-header h2 {
+  margin-bottom: 0%;
+}
+}
```
Comment (alex-latham) [inline src/containers/Faq/Faq.css:45]: The CSS gradient is so mesmerizing as the user scrolls down. I like it.

File: src/containers/Home/Home.js [hunk 3]
```diff
@@ -42,14 +59,6 @@ const Home = ({ setCurrentProfile, user }) => {
         {error &&
           <p className="error">{error}</p>
         }
-        {/* <input
```
Comment (J-Poulter) [inline src/containers/Home/Home.js:45]: Good catch taking this out before our final version

File: src/containers/Home/Home.js [hunk 4]
```diff
@@ -28,6 +27,24 @@ const Home = ({ setCurrentProfile, user }) => {
...
+      }
+      return response.json()
+    })
+    .then(response => loadSolarFaqs(response))
+    .catch((error) => {
+      window.alert(`Server Error. It's not your fault the error is: ${error}`)
+    })
+  }
+
+  useEffect(() => {
```
Comment (J-Poulter) [inline src/containers/Home/Home.js]: Learned a lot about useEffect and the different ways it has to be structured depending on how it's being used.

**Pull request merged**